src/pages/Swap/index.tsx

```diff
@@ -677,15 +677,15 @@ export default function Swap() {
                         }
                       >
                         <AutoRow justify="space-between" style={{ flexWrap: 'nowrap' }} height="20px">
-                          <span style={{ display: 'flex', alignItems: 'center' }}>
+                          <ThemedText.SubHeader width={'100%'} textAlign="center">
```

Comment:
I know that it's super inconsistent right now throughout the codebase, but I'll be sending a PR later today to switch to plain strings instead of `{'100%'}` (should be `"100%"`). Feel free to update if you have time. If not - my PR will most likely run an update globally on the entire code.

Reply:
I have changed it to plain string, @grabbou. Cheers